.github/workflows/gh-actions.yml

```diff
@@ -125,3 +125,85 @@ jobs:
           $RUNNER -q --jerry-tests
           --buildoptions=--compile-flag=-fsanitize=undefined,--compile-flag=-m32,--compile-flag=-fno-omit-frame-pointer,--compile-flag=-fno-common,--debug,--system-allocator=on,--linker-flag=-fuse-ld=gold
           --skip-list=parser-oom.js,parser-oom2.js
+
+  Linux_ARMv7l_Tests:
+    runs-on: ubuntu-latest
+    env:
+      RUNTIME: qemu-arm-static
+      TIMEOUT: 300
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo apt update
+      - run: sudo apt install gcc-arm-linux-gnueabihf libc6-dev-armhf-cross qemu-user-static
+      - run: >-
+          $RUNNER --jerry-tests -q
```

Comment:
I personally prefer to have -q / --quiet as the first argument (as it was in the travis configuration). But that is somewhat subjective.

Reply:
Personally I prefer it as the last argument, but I can deal with both options. The choice is on you.

Reply:
Ignore it, I checked the usage elsewhere and `-q` is always the first argument.